jiratools/example_config.py

```diff
@@ -0,0 +1,29 @@
+"""Build Example Config."""
+import argparse
+import os
+import shutil
+import sys
+
+from jgt_common import error_if
+
+from .utils import CONFIG_FILENAME, SAMPLE_CONFIG_FILENAME
+
+
+def cli_example_config(args: argparse.Namespace) -> None:
+    """Build example config."""
+    error_if(
+        not os.path.exists(SAMPLE_CONFIG_FILENAME),
+        message="Missing example config file: {}".format(SAMPLE_CONFIG_FILENAME),
+    )
+
+    prefix = "Copying" if args.install else "Would copy"
+    print("{} {} to {}".format(prefix, SAMPLE_CONFIG_FILENAME, CONFIG_FILENAME))
+    if args.install:
```

Comment:
Whatever the group decides, this would need to be consistent across all functions that accept `args`.

Reply:
Since these are passed to cli-specific helper functions throughout, I decided to pass the full `args` object to each to allow us simpler future flexibility if we find a reason to tweak our args, but I'm open to further "votes" to refactor, if that's what we want.

Reply:
I"m on the edge of that coin, but I come down just slightly on the side of passing in the full `args`.

Reply:
I'm pro passing the full args fwiw

Reply:
I agree with Lewis on this FWIW.  I get that its easier in the future if the args is already there, but only passing what you need makes it more clear what is _currently_ being used IMO.